recipes/libnftnl/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake
+
+class LibnftnlTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+    test_type = "explicit"
```

Comment:
```suggestion
    generators = "CMakeDeps", "CMakeToolchain"
```
No longer needed in Conan v2.